src/content/docs/more-integrations/open-source-telemetry-integrations/dbt-cloud/dbt-cloud-integration.mdx

```diff
@@ -0,0 +1,373 @@
+* git clone https://github.com/newrelic-experimental/newrelic-dbt-cloud-integration.git
+</Step>
+
+<Step>
+#### Cd into the airflow directory
+```bash
+cd newrelic-dbt-cloud-integration/airflow
+```
+</Step>
+
+<Step>
+#### Initialize Docker Compose
+```bash 
+docker-compose up airflow-init
+```
+</Step>
+
+<Step>
+#### Run Docker Compose
+```bash
+docker-compose up
+```
+</Step>
+
+<Step>
+  Launch the Airflow UI. http://localhost:8080
+</Step>
+<Step>
+#### Create the five Airflow connections needed for the DAG 
+
+**dbt_cloud_admin_api:** Allows us to connect to the dbt cloud admin api with SimpleHttpHook
+* Type: http
+* Host: https://cloud.getdbt.com/api/v2/accounts/ACCOUNT_ID/ (Replace ACCOUNT_ID with your dbt Cloud account id)
+* Password: Your [dbt cloud api token (Profile settings) or a service account token](https://docs.getdbt.com/docs/dbt-cloud-apis/user-tokens#user-tokens)
+
+**dbt_cloud_discovery_api:** Allows us to connect to the dbt discovery API
+* Type: http
+* Host: https://metadata.cloud.getdbt.com/graphql
+* Password: [Dbt cloud service account token](https://docs.getdbt.com/docs/dbt-cloud-apis/user-tokens#user-tokens)
+
+**nr_insights_insert:** Allows us to upload custom events to NR1
+* Type: http
+* Host: https://insights-collector.newrelic.com/v1/accounts/ACCOUNT_ID/events (Replace ACCOUNT_ID with your account id) 
+* Password: Your [NR Insights Insert API Key](https://one.newrelic.com/admin-portal/api-keys/insightkeys) 
+
+**nr_insights_query:** Allows us to query NR1 custom events
+* Type: http
+* Host: https://insights-api.newrelic.com/v1/accounts/ACCOUNT_ID/query (Replace ACCOUNT_ID with your account id) 
+* Password: Your [NR Insights Query API Key](https://one.newrelic.com/admin-portal/api-keys/insightkeys) 
+
+**SNOWFLAKE:** Allows us to query snowflake for failed test rows. There are [many ways](https://airflow.apache.org/docs/apache-airflow-providers-snowflake/stable/connections/snowflake.html) to configure a snowflake connection. 
+To configure using a private key pair, fill in the following attributes. 
+* Type: Snowflake
+* Login: Your Snowflake username
+* Account: Your Snowflake account 
+* Warehouse: Your Snowflake warehouse 
+* Role: Your Snowflake role. The role must have access to all the DBs used in dbt Cloud to get all failed test rows. 
+* Private Key Text: The full private key used for this connection.
+* Password: Pass phrase for the private key if it is encrypted. Blank if it is unencrypted.  
+</Step>
+<Step>
+Enable the new_relic_data_pipeline_observability_get_dbt_run_metadata2 DAG
+</Step>
+</Steps> 
+
+</Collapser>
+</CollapserGroup>
+
+## Find your data [#find-data]
+
+This integration creates three custom events in NR1
+
+<CollapserGroup>
+<Collapser
+    id="collapser-1-in-group-2"
+    title="dbt_job_run"
+>
+
+**dbt_job_run:** Provides metadata and the status of all completed runs. This event does not include any data about models, snapshots, seeds and tests. Attributes include:
```

Comment:
Is this an event data-type that the integration collects specific to this set up?

Reply:
Yes, we use an Airflow DAG to create three custom event types in NR1.

Reply:
To be more clear, the only data in this integration is the three custom events. It is different than typical APM integrations which focus on metrics and spans. In our case, we mostly care about discrete events (when something has failed, in particular)